src/ebonite/runtime/server/base.py

```diff
@@ -111,7 +111,7 @@ def _execute_method(interface: Interface, method: str, request_data, ebonite_id:
         except (ExecutionError, SerializationError) as e:
             raise MalformedHTTPRequestException(e.args[0])
 
-        if hasattr(result, 'read'):
+        if isinstance(result, bytes):
             rlogger.debug('Got response for [%s]: <binary content>', ebonite_id)
             return result
```

Comment:
You sure this works?

Reply:
I've checked `binary_payload` example for both servers (Flask and aiohttp)